client/src/pages/dashboard/navbar.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import WorkOutlineIcon from "@material-ui/icons/WorkOutline";
+import "./styles/nav.scss";
+
+const profilePicImage =
+  "https://i2-prod.mirror.co.uk/incoming/article10883656.ece/ALTERNATES/s615b/PROD-Lost-In-Space-Anniversary-party.jpg";
+
+export default function Navbar() {
+  return (
+    <AppBar position="static">
```

Comment:
This should be sticky so that it stays in place when the user scrolls down

Reply:
Switched to sticky.